relay/kusama/Cargo.toml

```diff
@@ -1,10 +1,12 @@
 [package]
-name = "staging-kusama-runtime"
-build = "build.rs"
-version = "1.0.0"
 authors.workspace = true
+build = "build.rs"
+description = "Runtime of the Kusama Network"
 edition.workspace = true
 license.workspace = true
+name = "staging-kusama-runtime"
```

Comment:
when/how we decided to call it staging? must have missed that.

Reply:
This was when [this ****](https://github.com/marlonhanks) used our crate names to publish his very obvious fraudulent clone of our repo, examples are https://crates.io/crates/xcm and his so-called [Dust programming language](https://github.com/dustlang/dust) (curiously, he didn't bother to rename all of the substrate source files from .rs to .ds).

So then we still had to publish our crates, even when crates.io has a clear policy of non-interference as they can't decide on the exact rules for name-squatting. In https://github.com/paritytech/polkadot-sdk/pull/1241, we finally pulled off the band-aid and renamed our crates, mostly by adding a `staging-` prefix to the names. I commented and said that I didn't quite care what the name was, as long as we really did commit to finally publishing our crates.